Documentation/user-guides/running-exporters.md

```diff
@@ -96,11 +96,91 @@ By default, the `PodMonitor` and `ServiceMonitor` objects include runtime metada
 | container    | __meta_kubernetes_pod_container_name | `{name}` of the container in the scraped pod                                  |
 | endpoint     | -                                    | `{spec.Port}` or `{spec.TargetPort}` if specified                             |
 
+### Relabeling and Metric Relabeling
+
+Prometheus Operator provides the same Prometheus capability to relabel a target before scrape or a metric before sample ingestion, below you can find examples of each use case and implementation for Service or Pod monitors.
+
+#### Relabeling
```

Comment:
It might be good to lead with a full ServiceMonitor/PodMonitor example to show where those snippets go.

Reply:
Done!
can you take a look @xiu ?

Reply:
LGTM with the 2 additional suggestions, I can't approve/merge it as I'm not a maintainer, let's see what the others have to say. Thanks!